packages/rustcat/PKGBUILD

```diff
@@ -5,7 +5,7 @@ pkgname=rustcat
 _binname=rcat
 pkgver=v2.0.0.r6.g95ac0c4
 pkgrel=1
```

Comment:
When a package is changed and the `pkgver` is not increased then pkgrel must be increased so `pkgrel=2`

Reply:
done